cmd/lnd/main.go

```diff
@@ -41,4 +41,9 @@ func main() {
 		_, _ = fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
+
+	exitCode, exited := shutdownInterceptor.GetExitCode()
+	if exited {
```

Comment:
You're right I don't think we don't need it since the status code is returned. I maybe misunderstood @ellemouton comment:
> probably need a bool to indicate that we have in fact exited... ie, GetExitCode can return an int along with a bool. If the bool is false, it means we have not exited yet.

My view is that if we wait for a lock to be released and retrieve the status code (0/1), it implies the process has already exited so no need for additional flag.